targets/simple_switch/tests/CLI_tests/testdata/indirect_res.out

```diff
@@ -1,5 +1,5 @@
 ????
-my_indirect_counter[0]=  BmCounterValue(packets=0, bytes=0)
+my_indirect_counter[0]= 0 bytes 0 packets
```

Comment:
can we use a more "structured" output, e.g. `(bytes=0, packets=0)`

Reply:
I guess once the transition is complete we can also switch back to the normal printing style?

Reply:
Not necessarily, I don't see a reason to change it again. There was no reason to have the Thrift type name there in the first place